.gitignore

```diff
@@ -0,0 +1 @@
+**/node_modules
```

Comment:
- [ ] [Optional] I can see that you added `node_modules` directory to `.gitignore` but node_modules directory already exists in your repo so Please remove it from your GitHub repo to make it more professional and clean.

### ⚠️I marked this as optional because the last reviewer forgot to mention it so please consider doing it⚠️